src/tree/ResolvableTreeItemBase.ts

```diff
@@ -23,6 +23,10 @@ export abstract class ResolvableTreeItemBase extends AzExtParentTreeItem impleme
         return Array.from(this.contextValues.values()).sort().join(';');
     }
 
+    public get description(): string | undefined {
```

Comment:
The `Proxy` object isn't handling this automagically?

Reply:
I'm pretty sure it's due to how `effectiveDescription` works in AzExtTreeItem.

https://github.com/microsoft/vscode-azuretools/blob/main/utils/src/tree/AzExtTreeItem.ts#L45-L47

Reply:
Gotcha. We should investigate if `iconPath` will also need it; seems like it might.